learn/advanced/asynchronous_operations.md

```diff
@@ -118,17 +139,95 @@ Had the task failed, the response would have included an `error` object:
 }
 ```
 
+If the task had been canceled while it was `enqueued` or `processing`, it would have the `canceled` status:
+
+```json
+{
+  "uid": 1,
+  "indexUid": null,
+  "status": "canceled",
+  "type": "taskCancelation",
+  "canceledBy": 5,
+  "details": {
+    "matchedTasks": 1,
+    "canceledTasks": 1,
+    "originalFilter": "?uid=1"
+  },
+  "error": null,
+  "duration": "PT1S",
+  "enqueuedAt": "2021-08-10T14:29:17.000000Z",
+  "startedAt": "2021-08-10T14:29:18.000000Z",
+  "finishedAt": "2021-08-10T14:29:19.000000Z"
+}
+```
+
+If a task is deleted after it was finished (`succeeded`, `failed`, or `canceled`), Meilisearch returns a [`task_not_found`](/reference/errors/error_codes.md#task-not-found) error.
```

Comment:
>Does this perhaps want to say that after tasks have been deleted, trying to retrieve them will return the above error?

Yes. Is it not clear? 🫣
I thought it would be better to add what types of tasks users can delete, hence the 
>finished (`succeeded`, `failed`, or `canceled`)

part